scheduler/src/cook/mesos/rebalancer.clj

```diff
@@ -266,31 +298,46 @@
     [(next-state state pending-job preemption-decision) preemption-decision]
     [state nil]))
 
+(defn taskless?
```

Comment:
"not-running?"

Or I could make it "running-now?"  and just filter by the negation of it where it's used?